### ISSUE TITLE
Roborazzi screenshots are different on Mac vs Linux

### ISSUE THREAD

**Issue by timothyfroehlich** [google/automotive-design-compose#1082]
IIRC this is a Robolectric issue. Should check into it and should probably disable the Roborazzi tasks on Mac.

**Comment by timothyfroehlich** [comment]
Yep, known issue. Can look at implementing what NowInAndroid did to recapture them.

**Comment by anshu10sep** [comment]
## ✅ Verified Fixed

**Fix commit:** [`d4917fc3`](https://github.com/google/automotive-design-compose/commit/d4917fc3) — Fix #1082: Disable roborazzi tasks on macOS (#2387)

**Validation:**
- `build-logic/src/main/kotlin/designcompose/conventions/roborazzi.gradle.kts` now checks `System.getProperty("os.name").lowercase().contains("mac")` and disables Roborazzi tasks on macOS
- Build confirmed successful on macOS arm64
- All unit tests pass

This issue should be closed.
